.pre-commit-config.yaml

```diff
@@ -6,25 +6,19 @@ repos:
     - id: go-vet
     - id: go-lint
  - repo: https://github.com/keep-network/pre-commit-hooks.git
-   rev: 63e729f
+   rev: v1.3.0
```

Comment:
Don't we want to leave `63e729f` as this is newer than the `v1.3.0` tag?

Reply:
Yes, you're right. Reverted here 79153f6